scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -4,605 +4,553 @@ import "GvsWarpTasks.wdl" as Tasks
+  File hapmap_resource_vcf_index = "gs://gcp-public-data--broad-references/hg38/v0/hapmap_3.3.hg38.vcf.gz.tbi"
+  File mills_resource_vcf = "gs://gcp-public-data--broad-references/hg38/v0/Mills_and_1000G_gold_standard.indels.hg38.vcf.gz"
+  File mills_resource_vcf_index = "gs://gcp-public-data--broad-references/hg38/v0/Mills_and_1000G_gold_standard.indels.hg38.vcf.gz.tbi"
+  File omni_resource_vcf = "gs://gcp-public-data--broad-references/hg38/v0/1000G_omni2.5.hg38.vcf.gz"
+  File omni_resource_vcf_index = "gs://gcp-public-data--broad-references/hg38/v0/1000G_omni2.5.hg38.vcf.gz.tbi"
+  File one_thousand_genomes_resource_vcf = "gs://gcp-public-data--broad-references/hg38/v0/1000G_phase1.snps.high_confidence.hg38.vcf.gz"
+  File one_thousand_genomes_resource_vcf_index = "gs://gcp-public-data--broad-references/hg38/v0/1000G_phase1.snps.high_confidence.hg38.vcf.gz.tbi"
+  File reference = "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.fasta"
+  File reference_dict = "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.dict"
+  File reference_index = "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.fasta.fai"
+
+  # fully-qualified table names
+  String fq_sample_table = "~{project_id}.~{dataset_name}.sample_info"
+  String fq_alt_allele_table = "~{project_id}.~{dataset_name}.alt_allele"
+  String fq_info_destination_table = "~{project_id}.~{dataset_name}.filter_set_info"
+  String fq_tranches_destination_table = "~{project_id}.~{dataset_name}.filter_set_tranches"
+  String fq_filter_sites_destination_table = "~{project_id}.~{dataset_name}.filter_set_sites"
+
+  call Utils.GetBQTableLastModifiedDatetime as SamplesTableDatetimeCheck {
+    input:
+      query_project = project_id,
+      fq_table = fq_sample_table,
+      service_account_json_path = service_account_json_path
+  }
+
+  call GetNumSamplesLoaded {
+    input:
+      fq_sample_table = fq_sample_table,
+      fq_sample_table_lastmodified_timestamp = SamplesTableDatetimeCheck.last_modified_timestamp,
+      service_account_json_path = service_account_json_path,
+      project_id = project_id
+  }
+
+  call Utils.SplitIntervals {
+    input:
+      intervals = interval_list,
+      ref_fasta = reference,
+      ref_fai = reference_index,
+      ref_dict = reference_dict,
+      scatter_count = scatter_count
+  }
+
+  scatter(i in range(length(SplitIntervals.interval_files))) {
+    call ExtractFilterTask {
+      input:
+        gatk_override            = gatk_override,
+        reference                = reference,
+        reference_index          = reference_index,
+        reference_dict           = reference_dict,
+        fq_sample_table          = fq_sample_table,
+        intervals                = SplitIntervals.interval_files[i],
+        fq_alt_allele_table      = fq_alt_allele_table,
+        excess_alleles_threshold = 1000000,
```

Comment:
we should make a ticket to just remove this "feature" from the filters… it didn't work out.  Maybe we need to confirm with Lee, I think it was his idea?